packages/twenty-front/src/modules/settings/accounts/components/SettingsNewAccountSection.tsx

```diff
@@ -0,0 +1,15 @@
+import { H2Title } from '@/ui/display/typography/components/H2Title';
+import { Section } from '@/ui/layout/section/components/Section';
+import { SettingsAccountsEmptyStateCard } from './SettingsAccountsEmptyStateCard';
+
+export const SettingsNewAccountSection = () => {
+  return (
+    <Section>
+      <H2Title
+        title="New account"
+        description="Connect a new account to your workspace"
+      />
+      <SettingsAccountsEmptyStateCard />
```

Comment:
As you mentioned here: https://github.com/twentyhq/twenty/issues/2894#issuecomment-1857595033
We want to display "Connect a Google account" instead of "No connected account" for this section.
Could you add an optional "label" prop to this component, that overrides the "No connected account" default label when defined, and then pass "Connect a Google account" as the label?